src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -239,18 +271,20 @@ void func_80A74398(Actor* thisx, GlobalContext* globalCtx) {
     }
 }
 
-s32 func_80A745E4(EnIk* this, GlobalContext* globalCtx) {
-    if (((this->unk_2FB != 0) || (this->actor.params == 0)) &&
+// Checks to see if action can be interrupted to block attack
+s32 EnIk_ChangeAction(EnIk* this, GlobalContext* globalCtx) {
+    if (((this->armorStatusFlag != 0) || (this->actor.params == 0)) &&
```

Comment:
```suggestion
    if (((this->armorStatusFlag != 0) || (this->actor.params == IK_TYPE_NABOORU)) &&
```
and in other checks against `params`